### PR TITLE
QA-820 os licenses split - the frontend part

### PR DESCRIPTION
**Pull request by mzedel**
The go-licenses format is not 100% matched, as I don't get the location of the license file from the license collection tool I get the `licenses.json` from - so this just links to the repository... I hope that's tolerable @oldgiova

### CODE REVIEW

File: frontend/pipeline.yml [hunk 1]
```diff
@@ -289,3 +294,57 @@ publish:frontend:docker:
...
+      artifacts: true
+  script:
+    - deno task --cwd frontend/scripts licenseFormatting --rootDir $(pwd)
+  artifacts:
+    when: on_success
+    expire_in: 1w
+    paths:
+      - licenses.md
+
+publish:frontend:licenses:docs-site:
```
Comment (oldgiova) [inline frontend/pipeline.yml]: Maybe is it better to move this to the main `.gitlab-ci.yml` and use a single `publish:license:docs-site` for both backend and frontend?

Like (here)[https://github.com/mendersoftware/mender-server/pull/220/files#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7R431] setting the two jobs:

```
  needs:
    - job: publish:backend:licenses
      artifacts: true
    - job: publish:frontend:licenses
      artifacts: true
```
And get an unified `LICENSE.md` file, also for a single mender-docs page?

**Review by oldgiova — COMMENTED** [review]
Thank you so much for your help! Just an observation: is it better, maybe, to use a single job for creating a single license file from both frontend and backend?

File: frontend/pipeline.yml (hunk 1 above)
Comment (mzedel) [reply frontend/pipeline.yml]: sure we can do that as well - I just carried on with the split until then, but I'll adjust 👌

Comment (oldgiova) [reply frontend/pipeline.yml]: Thanks!

**Review by oldgiova — APPROVED** [review]
Nice! Thanks for your help!

**Pull request merged**